README.md

```diff
@@ -20,20 +20,47 @@ Only the latest nightly build is kept.
 Prerequisites
 -------------
 
-#### *nix building
+### *nix building
 
+###### Minimum required packages:
 ```
-g++ make pkg-config libgtk2.0-dev libcanberra-gtk-module libgl-dev libasound2-dev
+g++ make pkg-config libgtk2.0-dev libcanberra-gtk-module libgl-dev libasound2-dev libao-dev libopenal-dev
 ```
 
-By default, GTK2 is used, but support for GTK3 is available. You will need to install the additional package `libgtk-3-dev` as well
-as specifying the command line option `hiro=gtk3` at compile time.
+###### GTK2 & GTK3
+By default, GTK2 is used, but support for GTK3 is available. You will need to install the additional package `libgtk-3-dev` as well as specifying the command line option `hiro=gtk3` at compile time.
 
-#### Windows building
+###### SDL2 for input
+If you would like to use SDL for input, you will need to install the following the `libsdl2-dev` and `libsdl2-0.0` packages and perform a clean build of ares. You should then be able to select SDL for input in the Settings > Drivers menu.   
+
+##### Building with clang
+
+clang++ is now the preferred compiler for ares as it is known to produce a higher performing executable. If it is detected, the build will default to building with clang. It is recommended to install the `clang` package. If you would like to manually specify a compiler, you can use the following option: `compiler=[g++|clang++]`
+
+### Windows building
 
 To build on Windows, using MSYS2 is recommended which can be download [here](https://www.msys2.org/). Follow the instructions
```

Comment:
This is probably still the minimal footprint toolchain for building ares, but it would be nice if we had instructions on how to replicate the official build configuration. I can help with this.

Reply:
Feel free to pass on instructions to me or submit an updated pull request if you have these available.